src/Pelagos/Bundle/AppBundle/Resources/views/SideBySide/index.html.twig

```diff
@@ -0,0 +1,159 @@
+                '//cdn.datatables.net/select/1.0.1/css/select.dataTables.min.css',
+                '//cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/css/select2.min.css',
+                '//cdnjs.cloudflare.com/ajax/libs/animate.css/3.3.0/animate.min.css',
+                '//cdnjs.cloudflare.com/ajax/libs/file-uploader/5.11.8/fine-uploader-new.min.css',
+                '//cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.2.0/css/ol.css',
+            ]
+        )
+    }}
+    
+    {% stylesheets
+        '@PelagosAppBundle/Resources/public/css/dataset-submission.css'
+        '@PelagosAppBundle/Resources/public/css/fileBrowser.css'
+        '@PelagosAppBundle/Resources/public/css/dataset-submission-uploader.css'
+        '@PelagosAppBundle/Resources/public/css/sidebyside.css'
+    %}
+        {{ add_css(asset_url) }}
+    {% endstylesheets %}
+    
+    {{
+        add_js(
+            [
+                '//cdnjs.cloudflare.com/ajax/libs/jquery-noty/2.3.5/packaged/jquery.noty.packaged.min.js',
+                '//cdn.datatables.net/1.10.7/js/jquery.dataTables.js',
+                '//cdn.datatables.net/select/1.0.1/js/dataTables.select.min.js',
+                '//cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.11.1/jquery.validate.min.js',
+                '//cdnjs.cloudflare.com/ajax/libs/openlayers/2.13.1/OpenLayers.js',
+                '//cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.2.0/build/ol.js',
+                '//cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/js/select2.min.js',
+                '//cdnjs.cloudflare.com/ajax/libs/datejs/1.0/date.min.js',
+                '//cdnjs.cloudflare.com/ajax/libs/spin.js/2.0.1/spin.min.js',
+                '//cdnjs.cloudflare.com/ajax/libs/file-uploader/5.11.8/jquery.fine-uploader/jquery.fine-uploader.min.js',
+                '//maps.google.com/maps/api/js?v=3&key=' ~ google_maps_api_key,
+            ]
+        )
+    }}
+    
+    {% javascripts
+        '@PelagosAppBundle/Resources/public/js/pelagosNoty.js'
+        '@FOSJsRoutingBundle/Resources/public/js/router.js'
+        '@PelagosAppBundle/Resources/public/js/datasetReview.js'
+        '@PelagosAppBundle/Resources/public/js/sidebyside.js'
+        '@PelagosAppBundle/Resources/public/js/geoviz.js'
+        '@PelagosAppBundle/Resources/public/js/mapWizard.js'
+        '@PelagosAppBundle/Resources/public/js/gMap.js'
+        '@PelagosAppBundle/Resources/public/js/common.js'
+        '@PelagosAppBundle/Resources/public/js/fileBrowser.js'
+    %}
+        {{ add_js(asset_url) }}
+    {% endjavascripts %}
+
+    {{ add_js(path('fos_js_routing_js', { callback: 'fos.Router.setData' })) }}
+{% endblock %}
+
+{% block body %}
+
+<div>
+<h1>UDI: <span class="udi-title"></span></h1>
+<table class="side-table">
+    <tr>
+        <td colspan="8" width="100%">
+            <label>UDI:</label>
+            <input name="udi" size="100">
+            <button id="get-versions-button" type="button">Get UDI</button>
```

Comment:
Can be disabled if no input is entered

Reply:
This can be done in a future improvement.